### ISSUE TITLE
draggable: false option not working with "setElements"

### ISSUE THREAD

**Issue by viliusle** [biati-digital/glightbox#257]
**Describe the bug**
`Draggable: false` option not working, if using with `setElements`. 

**Post the code you are using**
```
const lightbox = GLightbox()
const allLightboxTriggers = document.querySelectorAll('.lightbox-trigger')
if (allLightboxTriggers) {
  allLightboxTriggers.forEach(function(trigger) {
    trigger.addEventListener('click', function(e) {
      e.preventDefault()
      let targetHref = this.getAttribute('href')
      lightbox.setElements([{'href': targetHref},  draggable: false])
      lightbox.open()
    })
  })
}
```

**Desktop:**
 - Browser [chrome]

**Comment by frankie-tech** [comment]
Hey there @viliusle,

It looks like there is an issue with the code you're using to set the element, you have

```js
lightbox.setElements([{ 'href': targetHref }, draggable: false]);
```

and it should be:

```js
lightbox.setElements([{ href: targetHref, draggable: false }]);
```

Object keys can't be strings unless you wrap them in brackets `[`/`]`. The object inside of it was also malformed since the `draggable: false` was not inside of it.

**Comment by viliusle** [comment]
I re-checked it again, and looks like even `data-draggable="false"` or my example with your fixes does not work. `draggable` is ignored.

**Comment by frankie-tech** [comment]
Would you be able to add a link to a CodePen? I'll try to do some testing on my own, but a CodePen with an isolated example would help out a lot.

**Comment by viliusle** [comment]
https://jsfiddle.net/xz3rt8dy/5/

**Comment by frankie-tech** [comment]
I got a chance to take a real look at this and can confirm the bug.

What I found interesting though was that `draggable` can be set on the constructor and work all the time.

So take your example fiddle. If you were to change the constructors of each one to include an options object like so:

```js
const shouldBeDraggable = {
  draggable: true
}
```

It would prevent dragging for all of the slides, even the ones added dynamically.

The documentation lists a different behavior, so I will investigate this more once I have free time again.
